v03_pipeline/lib/misc/vets.py

```diff
@@ -0,0 +1,40 @@
+import hail as hl
+
+PASS_FILTER = 'PASS'  # noqa: S105
+VETS_SNP_CUTOFF = 0.997
+VETS_INDEL_CUTOFF = 0.99
+VETS_SNP_FILTER = 'high_CALIBRATION_SENSITIVITY_SNP'
+VETS_INDEL_FILTER = 'high_CALIBRATION_SENSITIVITY_INDEL'
+
+
+def annotate_vets(mt: hl.MatrixTable) -> hl.MatrixTable:
+    if not hasattr(mt, 'info') or not hasattr(mt.info, 'CALIBRATION_SENSITIVITY'):
+        return mt
+    return mt.annotate_rows(
+        filters=hl.bind(
+            lambda is_snp, split_cs: (
+                hl.case()
+                .when(
+                    is_snp & (split_cs > VETS_SNP_CUTOFF),
+                    hl.if_else(
+                        hl.is_defined(mt.filters),
+                        mt.filters.add(VETS_SNP_FILTER).remove(PASS_FILTER),
```

Comment:
would Hail's set remove throw an error if 'PASS' isn't present? I know sometimes callsets can be inconsistent if it's passed as 'PASS' versus just None

Reply:
```
In [5]: hl.set(['abc']).remove('10').remove('abc').remove('s').show()
+----------+
| <expr>   |
+----------+
| set<str> |
+----------+
| {}       |
+----------+

```

nope! we're all good.